### PR TITLE
Fix `NoneType` error when searching for /31 prefixes

### LINKED ISSUE #269: searching for /31 prefixes throws an error with v1.0.0b3
### Environment
* Python version: 3.7.3
* Nautobot version: 1.0.0b3

### Steps to Reproduce
1. search for a /31 prefix such as 10.150.255.2/31
2. click the Search button
3. Server Error appears:

> <class 'TypeError'>
> 
> cannot convert 'NoneType' object to bytes
> 
> Python version: 3.7.3
> Nautobot version: 1.0.0b3

### Expected Behavior
Search results with the /31 prefix

### Observed Behavior
Server Error appears:

> <class 'TypeError'>
> 
> cannot convert 'NoneType' object to bytes
> 
> Python version: 3.7.3
> Nautobot version: 1.0.0b3
> 

This was working correctly in Nautobot version: 1.0.0b2

### LINKED ISSUE #286: Error when searching for IP without prefix
### Environment
* Python version: 3.8.5
* Nautobot version: 1.0.0b3

When I use search field in the top right most corner (in navbar nex to user) and I search only by IP without prefix, I get the following error:
image

Full trace:
```
Traceback (most recent call last):
  File "/opt/nautobot/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/opt/nautobot/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/nautobot/lib/python3.8/site-packages/django/views/generic/base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/nautobot/lib/python3.8/site-packages/django/views/generic/base.py", line 98, in dispatch
    return handler(request, *args, **kwargs)
  File "/opt/nautobot/lib/python3.8/site-packages/nautobot/core/views/__init__.py", line 159, in get
    filtered_queryset = filterset({"q": form.cleaned_data["q"]}, queryset=queryset).qs
  File "/opt/nautobot/lib/python3.8/site-packages/django_filters/filterset.py", line 243, in qs
    qs = self.filter_queryset(qs)
  File "/opt/nautobot/lib/python3.8/site-packages/django_filters/filterset.py", line 230, in filter_queryset
    queryset = self.filters[name].filter(queryset, value)
  File "/opt/nautobot/lib/python3.8/site-packages/django_filters/filters.py", line 779, in __call__
    return self.method(qs, self.f.field_name, value)
  File "/opt/nautobot/lib/python3.8/site-packages/nautobot/ipam/filters.py", line 183, in search
    broadcast__gte=bytes(query.broadcast),

Exception Type: TypeError at /search/
Exception Value: cannot convert 'NoneType' object to bytes
```

Searching with prefix works:
image

### Steps to Reproduce
1. Search IP without prefix in the navbar box
image

### Expected Behavior
IP searched, results showed

### Observed Behavior
TypeError

### PR DESCRIPTION
**Pull request by jathanism**
### Fixes

- Fixes #269
- Fixes #286 

- Also applied this to `AggregateFilterSet.search`
- Also added `field_order` to `AggregateFilterForm` since the search box was showing in the middle of the fields vs. top
- Added missing filter tests for `search` for both models
- Some minor code cleanup (removed trailing `,` from some object creations

### CODE REVIEW

**Comment by glennmatthews** [comment]
Does this also fix #286 by any chance?

File: nautobot/ipam/tests/test_filters.py [hunk 1]
```diff
@@ -497,10 +504,16 @@ def setUpTestData(cls):
             vlan=vlans[2],
             role=roles[2],
             status=status_map["reserved"],
-        ),
+        )
         Prefix.objects.create(prefix="10.0.0.0/16", status=status_map["active"]),
         Prefix.objects.create(prefix="2001:db8::/32", status=status_map["active"]),
```
Comment (glennmatthews) [inline nautobot/ipam/tests/test_filters.py:509]: Couple more unneeded `,` here as long as you're fixing things. :-)

**Comment by jathanism** [comment]
> Does this also fix #286 by any chance?

Looks like it does. Thanks.

File: nautobot/ipam/tests/test_filters.py [hunk 2]
```diff
@@ -268,37 +269,43 @@ def setUpTestData(cls):
...
             tenant=tenants[1],
             date_added="2020-01-05",
-        ),
+        )
+
+    def test_search(self):
+        Aggregate.objects.create(prefix="10.150.255.0/31", rir=self.rirs[0])
+        Aggregate.objects.create(prefix="10.150.255.2/31", rir=self.rirs[0])
+        params = {"q": "10.150.255.2/31"}
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 1)
```
Comment (glennmatthews) [inline nautobot/ipam/tests/test_filters.py:308]: Mind also adding a search for `"10.150.255.0"` and/or `"10.150.255.2"` to demonstrate the expected behavior after fixing #286? Same request on the other test case below.

**Review by glennmatthews — APPROVED** [review]

**Pull request merged**